src/host/services/NMEA2000Service.cpp

```diff
@@ -39,7 +39,7 @@
 static NMEA2000Service *handlerContext;
 
 static void handler(const tN2kMsg &msg) {
-  DEBUG("Received N2K Message with pgn: %i", msg.PGN);
+  //DEBUG("Received N2K Message with pgn: %i", msg.PGN);
```

Comment:
I agree this message is annoying and we should have a way to disable it. I think commenting it is ok until we have a better way.